docs/how_to_setup_object_storage.md

```diff
@@ -0,0 +1,53 @@
+# How to Setup Object Storage
+
+This is a guide about how to setup object storage when format a volume. Different object storage may has different option value. Check the specific object storage for your need.
+
+## Access key and secret key
+
+For authentication, the access key and secret key are needed. You could specify them through `--access-key` and `--secret-key` options. Or you can set `ACCESS_KEY` and `SECRET_KEY` environment variables.
```

Comment:
Mention AMI_Role, for example, s3 and oss. When the machine can access S3 or OSS using role, we don't need to specify them.